app/controllers/manifests_controller.rb

```diff
@@ -3,6 +3,6 @@ def new
   end
 
   def index
-    @manifests = Manifest.all
+    @manifests = Manifest.authorized_search({public: true}).records.to_a
```

Comment:
should we have some type of feature spec to confirm the changed behavior of this controller?